### DISCUSSION TITLE
🔗 Ability to turn AI Feature Off (Duplicate of #22796)

### DISCUSSION THREAD

**Discussion by ghost** [elementor/elementor#23263] [Feature Request]
### Prerequisites

- [X] I have searched for similar features requests in both open and closed tickets and cannot find a duplicate.
- [X] The feature is still missing in the latest stable version of Elementor ( Elementor Pro )

### What problem is your feature request going to solve? Please describe.

Issue:
After upgrading to the latest version of Elementor and Elementor Pro, we experienced a massive slowdown for all of our sites (50+ in a multisite build) in regard to loading and editing with Elementor. Not only that, all of our sites are now experiencing load time performance issues.

In an effort to find the offending application, we turned off all plugins, themes and turned them back on one-by-one. Upon completion of our testing, we discovered that the new versions of Elementor and Elementor Pro were causing the extreme latency issues.

To combat this, we turned off almost all Experimental Features which improved our "Edit with Elementor" time from 32 seconds to 13 seconds which is an improvement, but not what it was before (about 6-8 seconds).

My conclusion, and please correct me if I am wrong, is that Elementor AI is greatly attributing to our latency issues by taxing our system resources.

### Describe the solution you'd like

My feature request, that may or may not fix my issue, would be to allow the administrator to disable the AI feature if it is not wanted, needed or used.

**Comment by hitsgee** [comment]
A setting for this would be great or even a filter we can add to the theme. Got many a client I do not want to have this functionality.

**Comment by kangarko** [comment]
Definitely should be toggleable.

**Comment by nicholaszein** [comment]
Hello there!

Thank you for your submission! 🙏 

### 🔗 🚀 There is already a similar request submitted by someone here #22796

Feel free to continue the conversation and cast your **⬆️ Upvote** there! 👉

### 🔍 Please, in the future, remember to check if there isn't any thread similar to yours already opened. 

Kind regards
